app/models/manager_refresh/inventory_object.rb

```diff
@@ -93,6 +97,18 @@ def self.add_attributes(inventory_object_attributes)
       end
     end
 
+    def ==(other)
```

Comment:
Yes, I know it is ineffective.

Used it in specs, where an inventory object is compared against a hash the of expected data to be assigned to the parser. Here for example: https://github.com/israel-hdez/manageiq/blob/c7b6bacc5f0252b11fb4390b122c12d15733d5fd/spec/models/manageiq/providers/hawkular/inventory/parser/middleware_manager_spec.rb#L57

Using it on specs is the only use case I can think of it for now. And you are right in that comparing against manager_uuid is all what is needed for real-life scenarios. May be renaming it to `hard_data_comparison` would be better?

Anyway, will remove it for now and use a helper method on specs.